README.md

```
@@ -1,92 +1,121 @@
-[TanStack Query](https://tanstack.com/query)
+   ```bash
+   pnpm install
+   ```
 
-[shadcn/ui](https://ui.shadcn.com)
+3. Set up environment variables:
 
-[TailwindCSS](https://tailwindcss.com)
+   ```bash
+   cp .env.example .env
+   ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)